setup.cfg

```diff
@@ -28,6 +28,16 @@ install_requires =
     bqplot-gl
     scikit-image
 
+[options.entry_points]
+glue.plugins =
+    histogram = glue_jupyter.bqplot.histogram:setup
+    image = glue_jupyter.bqplot.image:setup
+    profile = glue_jupyter.bqplot.profile:setup
+    scatter = glue_jupyter.bqplot.scatter:setup
```

Comment:
These plugin names might conflict with the ones from glue-core - the entry point names should be unique across the glue ecosystem.